### PR TITLE
fix to much power on non perfect OC

### PR DESCRIPTION
**Pull request by botn365**
now it should not consume to much power with non perfect OC and going past the tick limit

### CODE REVIEW

File: src/main/java/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_MultiBlockBase.java [hunk 1]
```diff
@@ -618,7 +618,17 @@ protected void calculateOverclockedNessMultiInternal(int aEUt, int aDuration, in
                 tempEUt <<= 2;//this actually controls overclocking
                 //xEUt *= 4;//this is effect of everclocking
                 mMaxProgresstime >>= ocTimeShift;//this is effect of overclocking
-                xEUt = mMaxProgresstime==0 ? xEUt >> ocTimeShift : xEUt << 2;//U know, if the time is less than 1 tick make the machine use less power
+                if (mMaxProgresstime <1)
+                {
+                    if(oldTime == 1)
```
Comment (repo-alt) [inline src/main/java/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_MultiBlockBase.java:623]: what is oldTime?

**Comment by botn365** [comment]
had a few git problems must have goten lost

File: src/main/java/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_MultiBlockBase.java (hunk 1 above)
Comment (repo-alt) [reply src/main/java/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_MultiBlockBase.java:624]: so, now if the overclock is perfect, and mMaxProgresstime was 3 in the previous step, oldTime is 3, and xEU is multiplied by 3

Comment (botn365) [reply src/main/java/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_MultiBlockBase.java:624]: yep 3 times the speed 3 times the enegry cost
stil not perfect but a lot beter now

Comment (repo-alt) [reply src/main/java/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_MultiBlockBase.java:624]: Not really. Time will remain 3 ticks, but energy will also increase 3 times

Comment (repo-alt) [reply src/main/java/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_MultiBlockBase.java:624]: ah, sorry, time will  be set to 1, ok

**Review by repo-alt — APPROVED** [review]

**Pull request merged**